cupy/random/generator.py

```diff
@@ -606,6 +606,9 @@ def _interval(self, mx, size):
 
         n = functools.reduce(operator.mul, size, 1)
 
+        if n is 0:
```

Comment:
```suggestion
        if n == 0:
```
This is more natural.

Reply:
agreed